tests/models/mamba2/test_modeling_mamba2.py

```diff
@@ -394,19 +478,25 @@ def test_attention_outputs(self):
                 list(self_attentions[0].shape[-3:]),
                 [self.model_tester.attention_num_heads, encoder_seq_length, encoder_key_length],
             )
+    
+    @slow
+    def test_model_from_pretrained(self):
+        model = Mamba2Model.from_pretrained("state-spaces/mamba2-130m")
+        self.assertIsNotNone(model)
```

Comment:
Let's look at the conversion stuff later.

Reply:
Sure, we can test it later.

Reply:
Can you outcomment it for now and make it as a TODO? so just pass/skip the test basically with a todo comment.